### DISCUSSION TITLE
Spree Subscriptions

### DISCUSSION THREAD

**Discussion by juanmiret** [spree/spree#10860] [General]
Hi! Just curious about what happened to Spree Subscriptions module? Cannot find either that or any reference to subscriptions in Documentation or the admin panel.

https://spreecommerce.org/features/   Here it says that Spree has built in subscriptions but couldn't find anything

**Comment by mradfaber** [comment]
Hi, we haven't open sourced any subscriptions code because they were all very specific for each use case.
There are 3 ways to implement repeat subscription business with Spree Commerce: https://spreecommerce.org/use-cases/subscription-ecommerce/ Depending on the use case one of them might be a better fit than others. Let me know your thoughts. Thanks!

**Reply by connyg** [reply]
Then you need to remove that USP from the website as there is no implementation to benefit from.
I am currently evaluating commerce engines, thought Spree was a better fit for the subscriptions to now learn that it is not - I wasted a few hours of time.
I think I will now go for Vendure as it is all node based what I would prefer.

**Reply by connyg** [reply]
Or could you provide me with the module for my use case outside of the public repo to have a starting point?

**Reply by badrange** [reply]
I arrived at this discussion after trying to find references to subscription in the source code. @mradfaber it would be helpful if you added a note to the use cases page explaining that subscriptions are not part of the project but may be "purchased" from you as a separate project (if that is the case?)

**Reply by badrange** [reply]
@connyg I am curious as to what solution you ended up with for your use case.

We're looking for an open source commerce solution that supports subscriptions and Vendure is one option but it isn't clear to me how or how well it support subscriptions/recurring payments. Cheers, Bernt

**Reply by connyg** [reply]
I have moved to Vendure for the moment for that reason, though I am currently tight on time I can spend on that new platform, so haven't progressed very much since then. A bit of progress on the frontend side, but no work on the backend yet.
